tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "ES2017",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+     "lib": [
+       "es2017",
+       "dom"
+     ],                             /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
```

Comment:
do we need to leave all these commented attributes in the file?

Reply:
I think this is the "all possible options" stock version of the file that typescript provides when you do 'tsc --init'. No need to keep it all, but it will be compiled away before production. Would you like it cleansed?

Reply:
if we are not using it, lets keep it clean and remove it. Anyway it will be in git history so anyone who wishes to refer can.

Reply:
fixed